force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -81,15 +83,14 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String removeDuplicateWords(String sentence) {
         // Split the sentence into words
-
         // Create a set to store the unique words
-
         // Add each word to the set
-
         // Join the unique words back into a sentence
-
         // Return the sentence with unique words
-        return null; // Replace null with the variable you used to store the result
+        List<String> lstString = sentence.split(' ');
+        Set<String> sStrings = new Set<String>(lstString);
+        String setConversion = String.join((Iterable<String>)sStrings, ' ');
+        return setConversion; // Replace null with the variable you used to store the result
```

Comment:
This is great. Here is my solution. You could even get this down to two lines if you want and I think not loose readability. 

  public static String removeDuplicateWords(String sentence) {
        Set<String> uniqueWords = new Set<String>(sentence.split(' '));
        return String.join(uniqueWords, ' ');
    }